### DISCUSSION TITLE
Next possible release date?

### DISCUSSION THREAD

**Discussion by uroybd** [koreader/koreader#12149] [Q&A]
Hello,

Any ETA for the next stable release?

**Comment by Frenzie** [comment]
There were various blockers but likely soonish.

**Reply by Commodore64user** [reply]
Is anything else missing at this point?

**Reply by Frenzie** [reply]
Time, also keep in mind you did quite a number on the translations, ;-)

**Reply by Commodore64user** [reply]
For the greater good 😊
